pkg/gardenlet/controller/backupbucket/reconciler.go

```diff
@@ -182,6 +183,7 @@ func (r *Reconciler) reconcileBackupBucket(
 	} else if extensionBackupBucket.Status.LastOperation == nil {
 		// if the extension did not record a lastOperation yet, don't update the status
```

Comment:
So, ideally, this shouldn't happen because of the predicate here,
https://github.com/gardener/gardener/blob/c11c86ae07d8ea784f5c41362cd41800f06bb3ed/pkg/controllerutils/predicate/predicate.go#L226-L228
But during testing, at some point, there was a nil pointer exception because the extension backupentry status didn't have lastoperation, so I had put this as a safety net.